usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -6883,7 +6884,11 @@ static int read_adapter_config_file(STDLL_TokData_t * tokdata,
     int whitemode = 0;
     int anymode = 0;
     int apqn_i = 0;             /* how many APQN numbers */
-    char *conf_dir = getenv("OCK_EP11_TOKEN_DIR");
+    /* Since the ep11 token config contains the path to libica that
+     * will later be dlopen()ed, we cannot use a token directory from
```

Comment:
I would call that the 'token config directory'. The 'token directory' usually is the directory where the token stores its data, e.g. /var/lib/opencryptoki/<token-name>/. 
Yes the environment variable name is misleading as well, but lets at least precise in the comment :-)

Reply:
done